lib/files.cf

```diff
@@ -778,7 +778,15 @@ bundle edit_line set_line_based(v, sep, bp, kp, cp)
       unless => strcmp( "true", $(edit.empty_before_use) );
 @endif
 
-    (cfengine_3_15|cfengine_3_16|cfengine_3_17|cfengine_3_18|cfengine_3_19|cfengine_3_20)::
+@if minimum_version(3.18)
+    !(cfengien_3_18_0|cfengine_3_18_1|cfengine_3_18_2)::
```

Comment:
`cfengine_3_18_0` -- the same typo is in the original commit.

Reply:
Hey, consistency!

Reply:
fixed, again!